v9 github codespace fails to build

### LVGL version

v9 current github

### What happened?

Followed to 3 clicks vscode github codespace procedure, build fails with multiple errors of the same type

/usr/local/emsdk/upstream/emscripten/cache/sysroot/include/bits/alltypes.h:182:15: error: Expected ,, instead got: int16_t
typedef short int16_t;
              ^
make[2]: *** [lvgl/CMakeFiles/lvgl.dir/build.make:1155: lvgl/CMakeFiles/lvgl.dir/src/draw/sw/blend/neon/lv_blend_neon.S.o] Error 1

### How to reproduce?

_No response_

--- a/.devcontainer/__lv_conf.h__
+++ b/.devcontainer/__lv_conf.h__
@@ -17,8 +17,6 @@
 #ifndef LV_CONF_H
 #define LV_CONF_H
 
-#include <stdint.h>
-
 #define LV_USE_DEV_VERSION 1
 
 /*====================

